ai/mistralai/v0/config/tasks.json

```diff
@@ -0,0 +1,429 @@
+          "default": 10,
+          "description": "Integer to define the top tokens considered within the sample operation to create new text (Note: The Mistral models does not support top-k sampling)",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUIOrder": 5,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Top K",
+          "type": "integer"
+        },
+        "top-p": {
+          "default": 0.5,
+          "description": "Float to define the tokens that are within the sample operation of text generation. Add tokens in the sample for more probable to least probable until the sum of the probabilities is greater than top-p (default=0.5)",
+          "instillAcceptFormats": [
+            "number",
+            "integer"
+          ],
+          "instillShortDescription": "Float to define the tokens that are within the sample operation of text generation",
+          "instillUIOrder": 6,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Top P",
+          "type": "number"
+        },
+        "safe": {
+          "description": "Safe generation mode",
+          "instillAcceptFormats": [
+            "boolean"
+          ],
+          "instillUIOrder": 7,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Safe",
+          "type": "boolean"
+        }
+      },
+      "required": [
+        "prompt",
+        "model-name"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Output",
+      "instillUIOrder": 0,
+      "properties": {
+        "text": {
+          "description": "Model Output",
+          "instillUIOrder": 0,
+          "instillFormat": "string",
+          "instillUIMultiline": true,
+          "title": "Text",
+          "type": "string"
+        },
+        "usage": {
+          "$ref": "#/$defs/chat-usage"
+        }
+      },
+      "required": [
+        "text"
+      ],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_TEXT_EMBEDDINGS": {
+    "instillShortDescription": "Turn text into a vector of numbers that capture its meaning, unlocking use cases like semantic search.",
+    "description": "An embedding is a list of floating point numbers that captures semantic information about the text that it represents.",
+    "input": {
+      "instillUIOrder": 0,
+      "properties": {
+        "model-name": {
+          "enum": [
+            "mistral-embed"
+          ],
+          "example": "mistral-embed",
+          "description": "The Mistral embed model to be used",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUIOrder": 0,
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
```

Comment:
Important
I think we also need instillCredentialMap for embedding.

Reply:
Fixed